qpc/tests/report/test_report_deployments.py

```diff
@@ -143,10 +139,9 @@ def test_deployments_report_as_csv(self, caplog, csv_file_path):
             with caplog.at_level(logging.INFO):
                 self.command.main(args)
                 assert messages.REPORT_SUCCESSFULLY_WRITTEN in caplog.text
-                with Path(csv_file_path).open("r", encoding="utf-8") as json_file:
-                    data = json_file.read()
-                    file_content_dict = json.loads(data)
-                assert get_report_csv_data == file_content_dict
+                with Path(csv_file_path).open("r", encoding="utf-8") as csv_file:
+                    data = csv_file.read()
+                assert get_report_response == data
```

Comment:
[optional] I'd keep the scope of `caplog.at_level` limited to line `140`. The rest of the test can run perfectly fine outside of that scope (or at least it should 🤔 )

Reply:
Good catch! I tightened up these scopes and amended the previous commit that touched these lines. Thanks!